apps/rotm/behaviours/skip-step.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  getValues(req, res, callback) {
```

Comment:
I initially had getValues setup like this
```
  getValues(req, res, callback) {
    super.getValues(req, res, (err) => {
...
```
but I couldn't get my test working, 
https://github.com/UKHomeOffice/rotm/pull/117/files#diff-ebc74498dd4afa561aae9e426d61ad54R1

Do you know what I did wrong?